nncf/experimental/torch/sparsity/movement/MovementSparsity.md

```diff
@@ -50,14 +50,14 @@ Please refer [optimum-intel](https://github.com/huggingface/optimum-intel/tree/m
 #### Known Limitation
 
 1. Movement sparsification only supports `torch.nn.Linear` layers.
-2. Automatic structured masking feature only supports **_BERT, Wav2Vec2 and Swin_**.
+2. Automatic structured masking feature supports **BERT, DistilBERT, RoBERTa, MobileBERT, Wav2Vec2, Swin, ViT, CLIPVisual** architectures defined by [Hugging Face&#39;s transformers](https://huggingface.co/docs/transformers/index). Other similar architectures may work, but support is not guaranteed.
```

Comment:
A minor comment about `CLIPVisual`. I am not sure what it means. Please rephrase.